### PR TITLE
fix numeric_static_map description

### LINKED ISSUE #1056: numeric_static_map vindex docs are incorrect
From the docs:
```
numeric_static_map	Functional Unique	JSON file statically mapping input string values to keyspace IDs
```
Looking at the code this is supposed to map numeric values to keyspace IDs. The docs need to be corrected for all versions (11.0 -> 15.0)
Link: https://vitess.io/docs/15.0/reference/features/vindexes/

### PR DESCRIPTION
**Pull request by harshit-gangal**
This PR fixes the description of numeric_static_map vindex description from string to numeric as input value expected.

Fixes: https://github.com/vitessio/website/issues/1056

### CODE REVIEW

**Review by systay — APPROVED** [review]

**Review by frouioui — APPROVED** [review]
LGTM!

File: content/en/docs/11.0/reference/features/vindexes.md [hunk 1]
```diff
@@ -201,24 +201,24 @@ Note: You can have `NULL` values for the primary vindex column, as long as that
...
-lookup | Lookup NonUnique | Lookup table non-unique values | No | Identify Row | No | Yes [only if](../vindexes/#ignore-nulls) | 20 | Any |
-lookup\_unique | Lookup Unique | Lookup table unique values | If unowned | Identify Row | No | Yes [only if](../vindexes/#ignore-nulls) | 10 | Any |
-null | Functional Unique | Always map to keyspace ID 0 | Yes | No | No | Yes | 100 | Any |
-numeric | Functional Unique | Identity | Yes | No | Yes | No | 0 | 64 bit or smaller numeric or equivalent type |
-numeric\_static\_map | Functional Unique | JSON file statically mapping input string values to keyspace IDs | Yes | No | No | No | 1 | Any |
-region\_experimental | Functional Unique | Multi-column prefix-based hash for use in geo-partitioning | Yes | Yes | No | No | 1 | String and numeric type |
+Name | Type | Description                                                                                                    | Primary | Multi-column | Reversible | Nullable | Cost | Data types |
+---- | ---- |----------------------------------------------------------------------------------------------------------------| ------- | ------------ | ---------- | -------- | ---- | ---------- |
```
Comment (deepthi) [inline content/en/docs/11.0/reference/features/vindexes.md:205]: These formatting changes don't seem to be required? The pages look the same before and after (checked preview against current vitess.io for 11.0 and 12.0).

**Review by deepthi — COMMENTED** [review]
Thank you 😄

File: content/en/docs/11.0/reference/features/vindexes.md (hunk 1 above)
Comment (harshit-gangal) [reply content/en/docs/11.0/reference/features/vindexes.md:205]: This happens in the IDE by itself as I changed the text below in the table content.

**Pull request merged**